notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2205,6 +2205,47 @@ father and daughter [Frank beloved his young daughter Hanna and did everything i
 coping with the death of a parent [Hanna had to come to grips with the fact that the soldiers shot her virus infected father to death and that he was not coming back.]
 
 
+movie: Equilibrium (2002)
+=========================
+
+:: Title
+Equilibrium
+
+:: Description
+Equilibrium is a 2002 American science fiction action film written and
+directed by Kurt Wimmer, and starring Christian Bale, Emily Watson, and Taye
+Diggs.
+
+Synopsis: An enforcement officer in a future city-state comes to question why
+feelings and artistic expression are outlawed and citizens take psychoactive
+drugs to suppress their emotions.
+
+:: Date
+2002-12-06
+
+:: References
+https://en.wikipedia.org/wiki/Equilibrium_(film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+totalitarian dystopia [The story is set in a future totalitarian state where feelings and artistic expression are outlawed and citizens are compelled by law to take emotion suppressing drugs.]
```

Comment:
Does drugged up dystopia also apply?

Reply:
I added **drugged up utopia** (revised the theme description to boot) and **conformist dystopia**.